### PR TITLE
Init service

### PR DESCRIPTION
**Pull request by GabrielCousin**
This PR aims at adding the fundamental things needed to pursue the development:
- the initializer will inject the service into routes (I won't do it for controllers finally)
- the service has been created (all polling modes are supported, flag values are tracked, some methods were added)

The local client will be added in a next PR as well as tests (local client may help to mock)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +32,88 @@ This addon includes a service which wraps a [ConfigCat](https://configcat.com/)
...
+```
+
+### Public methods and properties
+
+| Name           | Type             | Description                                         |
+| -------------- | ---------------- | --------------------------------------------------- |
+| `flags`        | tracked property | Returns available feature-flags and values          |
+| `identifyUser` | method           | Identifies user and update feature-flags values     |
+| `update`       | method           | Updates feature-flags values                        |
+| `dispose`      | method           | Releases everything related to the ConfigCat client |
```
Comment (tachun) [inline README.md:115]: Maybe add some example for this part?

**Review by tachun — APPROVED** [review]
🔥

File: README.md (hunk 1 above)
Comment (GabrielCousin) [reply README.md:151]: 👍

**Comment by GabrielCousin** [comment]
Removing anything related to service injection because of this https://emberjs.github.io/rfcs/0680-implicit-injection-deprecation.html

**Pull request merged**